.github/workflows/deploy-enterprise.yml

```diff
@@ -62,10 +62,10 @@ jobs:
             cost_profile: DEFAULT
             job_files: >-
               job_spec/ARIA_RAIDER.yml
-              job_spec/INSAR_ISCE_TEST.yml
+              job_spec/INSAR_ISCE.yml
             instance_types: m6id.xlarge,m6id.2xlarge,m6id.4xlarge,m6id.8xlarge,m6idn.xlarge,m6idn.2xlarge,m6idn.4xlarge,m6idn.8xlarge
-            default_max_vcpus: 1000
-            expanded_max_vcpus: 1000
+            default_max_vcpus: 1000  # Max: 10316
+            expanded_max_vcpus: 1000  # Max: 10316
```

Comment:
@cmarshak we can go up to 10,000 vCPUs in the hyp3-tibet-jpl deployment. Do you want to go all the way or stick with 1K for now?

Reply:
Leave as is until we know more about stability of M family on spot market. We are still spending 2k a day at this throughput. Thats more than fine.